### ISSUE TITLE
A11y -  Header - a11y search bar

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1364]
## User story

Search bar in header should be a11y

## Acceptance criteria

There are a couple low priority aspects to fix

- [ ] Search bar tab order wrong, it should receive focus before social links (prio: 1)
- [ ] crosscheck that search input is a11y: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/search_role (prio: 2)
